relayer/vector.yaml

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
I don't think we need this logging stuff in the container

Reply:
https://github.com/Snowfork/snowbridge/pull/1269/commits/1175db4498249bb05a86ae711bcde1f6be0f50ea